examples/run_Hahn_echo.py

```diff
@@ -90,26 +93,23 @@
     echo_data.squeeze()
     echo_data.set_units("t2", "s")
     fl.next("Raw - time")
-    fl.image(
-        echo_data.C.mean("nScans"))
+    fl.image(echo_data)
     echo_data.reorder("t2", first=False)
     for_plot = echo_data.C
-    for_plot.ft('t2',shift=True)
-    for_plot.ft(['ph1'], unitary = True)
-    fl.next('FTed data')
-    fl.image(for_plot.C.mean("nScans")
-    )
+    for_plot.ft("t2", shift=True)
+    for_plot.ft(["ph1"], unitary=True)
+    fl.next("FTed data")
+    fl.image(for_plot)
 else:
     if config_dict["nScans"] > 1:
         echo_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
     echo_data.rename("t", "t2")
     fl.next("Raw - time")
-    fl.image(
-        echo_data.C.mean("nScans"))
+    fl.image(echo_data)
     echo_data.reorder("t2", first=False)
     for_plot = echo_data.C
-    for_plot.ft('t2',shift=True)
-    fl.next('FTed data')
+    for_plot.ft("t2", shift=True)
+    fl.next("FTed data")
     fl.image(for_plot)
 echo_data.name(config_dict["type"] + "_" + str(config_dict["echo_counter"]))
 echo_data.set_prop("postproc_type", "proc_Hahn_echoph")
```

Comment:
I would say that, in general, we should not FT before saving the data --> that's something that we leave to the postproc scripts

Reply:
this is not FTing the saved data it makes a copy for plotting purposes and FTs that